packaging/amazon-linux-ami/ecs-init.spec

```diff
@@ -27,11 +27,11 @@ Source1:        ecs.conf
 
 BuildRequires:  golang >= 1.7
 
-Requires:       docker = 17.03.2ce
+Requires:       docker >= 17.03.2ce
 Requires:       upstart
 Requires:       iptables
 Requires:       procps
-Requires:       dhclient
+Requires(post): docker >= 1.6.0
```

Comment:
Not sure this makes sense anymore given the Requires on docker >= 17.03.2ce.

Reply:
Yeah, this was a vestige of something long ago and should stay removed.

Reply:
Since these are just revert commits, we can probably leave this for now.